packages/assistify-help-request/server/hooks/searchHooks.js

```diff
@@ -0,0 +1,21 @@
+/* global SystemLogger */
+function onSpotlighStartHandler(item) {
+	SystemLogger.debug('onSpotlightStart', item.text);
+}
+
+function onSpotlighEndHandler(item) {
+	SystemLogger.debug('onSpotlightEnd', item.text);
+	if (item.result && item.result.rooms) {
+		const subscripedChannels = RocketChat.models.Subscriptions.findByUserId(Meteor.userId()).fetch();
+		const publicChannelsNotContaingUser = RocketChat.models.Rooms.findAllPublicNotContainingUserName(Meteor.user().username).fetch();
+		const accessibleRooms = subscripedChannels.concat(publicChannelsNotContaingUser);
```

Comment:
@lobo87 I don't think this works out as two different entities. Probably:

```javascript
const accessibleRoomIds = subscribedChannels.map((subscription)=>subscription.rid)
.concat(publicChannelsNotContaingUser.map((room)=>room._id));
```

Reply:
@mrsimpson the idea is/was that for now I keep the full room object until we really know, what the endpoint needs.